docs/podman-pod-rm.1.md

```diff
@@ -0,0 +1,39 @@
+% podman-pod-rm "1"
+
+## NAME
+podman\-pod\-rm - Remove one or more pods
+
+## SYNOPSIS
+**podman rm** [*options*] *container*
+
+## DESCRIPTION
+**podman pod rm** will remove one or more pods from the host.  The pod name or ID can be used. The -f option stops all containers then removes them before removing the pod. Without the -f option, a pod cannot be removed if it has attached containers.
```

Comment:
Can we backtick `-f` in both occurrences here?

Also "stops all containers then removes them" -> "stops all containers and then removes them".

Also perhaps "attached" -> "associated", to avoid overloading "attached" which we already use for the unrelated terminal operations.